schema/crdb/dbinit.sql

```diff
@@ -3082,6 +3082,11 @@ CREATE TABLE IF NOT EXISTS omicron.public.blueprint (
     -- now gone.
     parent_blueprint_id UUID,
 
+    -- identifies the latest internal DNS version when blueprint planning began
+    -- XXX-dap when it comes time for the migration, make sure to update
```

Comment:
When this PR has approval, I'll implement the db migration part.  That part needs to make sure we provide a value for the `internal_dns_version` field for existing rows.  I'm planning to set it to 1.  That should be the current DNS version deployed everywhere.  It should also be safe even if it weren't -- having one that's too old is always safe.  It just means you won't be able to execute that blueprint and you need to re-plan to make a new one.